SU-HKKU/cardinal_service_profile
# CSD-182 Used flag module to provide a way for users to favorite opportunities

# READY FOR REVIEW

# Summary
- Provide a way for users to save & favorite different opportunities.
- User dashboard to view those favorited items
- User profile fields to edit their profile.

# Need Review By (Date)
- 7/1

# Urgency
- medium

# Steps to Test
1. `composer require SU-HKKU/cardinal_service_profile:dev-CSD-182`
1. `blt drupal:sync --site=cardinalservice`
1. when logged out, go to view the list of opportunities
1. click the hear to favorite an item
1. it should push you to log in and then kick you back to the list
1. it should then have that item favorited and display "Saved" on the item
1. go to `/user` and see you now have items count other than 0
1. click on the "Opportunities" and verify you have a saved item.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
